docker/dev_vpp_agent/Dockerfile

```diff
@@ -7,22 +7,18 @@ ARG VPP_COMMIT="xxx"
 RUN apt-get update && apt-get install -y \
     sudo wget git build-essential gdb vim nano python \
     iproute2 iputils-ping inetutils-traceroute libapr1 supervisor \
-    default-jre default-jdk telnet netcat software-properties-common
+    default-jre default-jdk telnet netcat \
+    golang-gogoprotobuf-dev software-properties-common net-tools
 
-RUN gcc --version  
 RUN add-apt-repository ppa:ubuntu-toolchain-r/test && \
     apt-get update && apt-get install -y gcc-7
 
-RUN mkdir /opt/vpp-agent
-RUN mkdir /opt/vpp-agent/dev
-RUN mkdir /opt/vpp-agent/plugin
-RUN mkdir /etc/supervisord
-
-WORKDIR /opt/vpp-agent/dev
+RUN mkdir /opt/vpp-agent && mkdir /opt/vpp-agent/dev  && \ 
+    mkdir /opt/vpp-agent/plugin && mkdir /etc/supervisord
```

Comment:
Command `mkdir` supports multiple directories as arguments. So this can be shortened to single command:
```Dockerfile
RUN mkdir /opt/vpp-agent /opt/vpp-agent/dev /opt/vpp-agent/plugin /etc/supervisord
```